r/R/dplyr-functions.R

```diff
@@ -109,6 +109,39 @@ nse_funcs$as.numeric <- function(x) {
   Expression$create("cast", x, options = cast_options(to_type = float64()))
 }
 
+# is.* type functions
+nse_funcs$is.character <- function(x) {
+  x$type_id() %in% Type[c("STRING", "LARGE_STRING")]
+}
+
+nse_funcs$is.numeric <- function(x) {
+  x$type_id() %in% c(2:12, 23:24)
```

Comment:
I would spell these out too with the Type enum, just so they're obvious

Reply:
Done in 9f7fa6e67b12913876b166493eb7e9024e93b306